api/server/sdk/volume_ops.go

```diff
@@ -811,6 +811,12 @@ func (s *VolumeServer) mergeVolumeSpecs(vol *api.VolumeSpec, req *api.VolumeSpec
 		spec.ExportSpec = vol.GetExportSpec()
 	}
 
+	// Xattr
+	if req.GetXattr() >= 0 {
```

Comment:
here, you want to use req.GetXattrOpt() referring to the in line 514 of api.proto. This will be nil if not provided. See the example above on line 801 on this file